collector.go

```diff
@@ -67,12 +122,33 @@ type ChunkedCollector struct {
 	started, stopped bool
 	stopChan         chan struct{}
 
+	queueSizeBytes  uint64
 	pendingBySpanID map[SpanID]Annotations
 
 	// mu protects pendingBySpanID, lastErr, started, stopped, and stopChan.
 	mu sync.Mutex
 }
 
+// NewChunkedCollector is shorthand for:
+//
+// 	c := &ChunkedCollector{
+// 		Collector:    c,
+// 		MinInterval:  500 * time.Millisecond,
+// 		FlushTimeout: 50 * time.Millisecond,
+// 		MaxQueueSize: 32 * 1024 * 1024, // 32MB
+// 		Log:          log.New(os.Stderr, "appdash: ", log.LstdFlags),
+// 	}
```

Comment:
After discussion on Slack me and @shurcooL decided it's best to keep this as-is for now, due to there being no better way to document that the code is equal (it's just as right to create a `&ChunkedCollector{...}` as it is to create a `NewChunkedCollector(...)`, users should not think that `NewChunkedCollector` does anything special in terms of initialization).

I'll keep thinking about this for future changes, though.